docs/en/inference/pipeline.md

```diff
@@ -143,6 +143,27 @@ response = pipe(prompts, gen_config=gen_config)
 print(response)
 ```
 
+- **An example for slora.**
+
+```python
+from lmdeploy import pipeline, GenerationConfig, PytorchEngineConfig
+
+backend_config = PytorchEngineConfig(session_len=2048,
+                                     adapters=dict(lora_name_1='chenchi/lora-chatglm2-6b-guodegang'))
+gen_config = GenerationConfig(top_p=0.8,
+                              top_k=40,
+                              temperature=0.8,
+                              max_new_tokens=1024)
+pipe = pipeline('THUDM/chatglm2-6b',
+                backend_config=backend_config)
+prompts = [[{
+    'role': 'user',
+    'content': '您猜怎么着'
```

Comment:
I am not sure if it is a good idea to use Chinese example in English docs.

Reply:
Do you have any other examples?

Reply:
Nope